src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -2272,49 +1259,146 @@ export default class Aircraft {
             return;
         }
 
+        this.updateAircraftTurnPhysics();
+
+        this.updateAltitudePhysics();
+
+        if (this.isOnGround()) {
+            this.trend = 0;
+        }
+
+        // SPEED
+        this.updateSpeedPhysics();
+
+        if (!this.position) {
+            return;
+        }
+
+        // Trailling
+        if (this.position_history.length === 0) {
+            this.position_history.push([
+                this.position[0],
+                this.position[1],
+                window.gameController.game_time() / window.gameController.game_speedup()
+            ]);
+            // TODO: this can be abstracted
+        } else if (abs((window.gameController.game_time() / window.gameController.game_speedup()) - this.position_history[this.position_history.length - 1][2]) > 4 / window.gameController.game_speedup()) {
+            this.position_history.push([this.position[0], this.position[1], window.gameController.game_time() / window.gameController.game_speedup()]);
+        }
+
+        // FIXME: is this ratio correct? is it 0.000514444 or 0.514444?
+        const scaleSpeed = this.speed * 0.000514444 * window.gameController.game_delta(); // knots to m/s
```

Comment:
@erikquinn can you verify this ratio?  I remember a while back seeing the second number listed in the fixme comment.  I just want to make sure we're using the correct number here.